### ISSUE TITLE
Issue running using Docker 

### ISSUE THREAD

**Issue by Isanderthul** [Manisso/fsociety#12]
when running the docker compose on a mac I get the following error:
```
➜  fsociety git:(master) docker-compose run --rm fsociety fsociety
Building fsociety
Step 1/7 : FROM python:2.7-slim
 ---> 4fd30fc83117
Step 2/7 : RUN apt-get update && apt-get install -qq -y --no-install-recommends     build-essential git sudo wget curl
 ---> Using cache
 ---> 0e445c2e14bc
Step 3/7 : RUN pip install requests
 ---> Using cache
 ---> c832916fdb65
Step 4/7 : RUN git clone https://github.com/Manisso/fsociety.git
 ---> Using cache
 ---> e3a090c4468f
Step 5/7 : RUN cd fsociety && chmod +x install.sh && ./install.sh
 ---> Using cache
 ---> a9768629ce92
Step 6/7 : RUN rm -rf fsociety
 ---> Using cache
 ---> ca6e7e2ccd5f
Step 7/7 : RUN fsociety
 ---> Running in 2d8420ae9065
TERM environment variable not set.
TERM environment variable not set.
TERM environment variable not set.
TERM environment variable not set.
TERM environment variable not set.

d88888b .d8888.  .d88b.   .o88b. d888888b d88888b d888888b db    db
88'     88'  YP .8P  Y8. d8P  Y8   `88'   88         88    `8b  d8'
88ooo   `8bo.   88    88 8P         88    88ooooo    88     `8bd8'
88        `Y8b. 88    88 8b         88    88         88       88
88      db   8D `8b  d8' Y8b  d8   .88.   88.        88       88
YP      `8888Y'  `Y88P'   `Y88P' Y888888P Y88888P    YP       YP

            }-----{+} Coded By Manisso {+}-----{
         }--------{+}  fb.me/dzmanisso {+}--------{
            }-----{+} Greetz To IcoDz  {+}-----{

   {1}--Information Gathering
   {2}--Password Attacks
   {3}--Wireless Testing
   {4}--Exploitation Tools
   {5}--Sniffing & Spoofing
   {6}--Web Hacking
   {7}--Private Web Hacking
   {8}--Post Exploitation
   {0}--INSTALL & UPDATE
   {99}-Exit

fsociety~# Traceback (most recent call last):
  File "/usr/share/doc/fsociety/fsociety.py", line 1808, in <module>
    menu()
  File "/usr/share/doc/fsociety/fsociety.py", line 131, in menu
    choice = raw_input("fsociety~# ")
EOFError: EOF when reading a line
ERROR: Service 'fsociety' failed to build: The command '/bin/sh -c fsociety' returned a non-zero code: 1
➜  fsociety git:(master)
```

**Comment by thehappydinoa** [comment]
I will look into this, thanks for reporting.

**Comment by cdoggyd** [comment]
I'm seeing the same error on my MacBook Pro running macOS 10.13.2 and Docker 17.09.1-ce.

**Comment by thehappydinoa** [comment]
Ok so I did some looking, I think this is happening bc raw_input does not work in the Python window in docker since python:2.7-slim is not a command line app there is no sys.stdin to read from. Any suggestions on what else we should use?

**Comment by thehappydinoa** [comment]
Fixed in #13
